From abd0cf15a612ecb26797f30fe23085940cc87ba6 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 26 Aug 2014 12:49:31 -0400
Subject: [tools] cpupower: Remove all manpages on make uninstall

Message-id: <1409057382-26432-21-git-send-email-prarit@redhat.com>
Patchwork-id: 89086
O-Subject: [RHEL7.1 PATCH BZ 1133893 20/31] cpupower: Remove all manpages on make uninstall
Bugzilla: 1133893
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1133893

commit 706f4c1100e7131f2e8aa9552daba52fbf6b97cb
Author: Ramkumar Ramachandra <artagnon@gmail.com>
Date:   Tue May 13 12:41:40 2014 +0200

    cpupower: Remove all manpages on make uninstall

    Signed-off-by: Ramkumar Ramachandra <artagnon@gmail.com>
    Signed-off-by: Thomas Renninger <trenn@suse.de>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Tony Camuso <tcamuso@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/power/cpupower/Makefile b/tools/power/cpupower/Makefile
index 3651db7..7b17ce7 100644
--- a/tools/power/cpupower/Makefile
+++ b/tools/power/cpupower/Makefile
@@ -295,8 +295,12 @@ uninstall:
  - rm -f $(DESTDIR)${libdir}/libcpupower.*
  - rm -f $(DESTDIR)${includedir}/cpufreq.h
  - rm -f $(DESTDIR)${bindir}/utils/cpupower
- - rm -f $(DESTDIR)${mandir}/man1/cpufreq-set.1
- - rm -f $(DESTDIR)${mandir}/man1/cpufreq-info.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower-frequency-set.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower-frequency-info.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower-set.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower-info.1
+ - rm -f $(DESTDIR)${mandir}/man1/cpupower-monitor.1
  - for HLANG in $(LANGUAGES); do \
   rm -f $(DESTDIR)${localedir}/$$HLANG/LC_MESSAGES/cpupower.mo; \
    done;
-- 
1.7.1